trace/README.rst

```diff
@@ -0,0 +1,12 @@
+OpenCensus - A stats collection and distributed tracing framework
```

Comment:
This file is for opencensus-trace, and the top level is for opencensus overall. I'll add sample usage after merging the tracers stuff.